src/logger/logger.hpp

```diff
@@ -0,0 +1,96 @@
+#ifndef WEBSERV_SRC_LOGGER_LOGGER_HPP
+#define WEBSERV_SRC_LOGGER_LOGGER_HPP
+
+#include "null_stream.hpp"
+#include <fstream>
+#include <iostream>
+
+class ILoggerHandler;
+
+enum LogLevel {
+  kInfo = 0,
+  kWarn = 1,
+  kError = 2,
+  kDebug = 3,
+};
```

Comment:
聞きたいこと
- kってなんかの略語？
- enum全部大文字のイメージあったけど、c++だとあんま関係ない？
　- e.g. nullptrとかもいう例もあるからいいのか

Reply:
- ドイツ語由来らしい。
>"c" was the tag for type "char", so it couldn't also be used for "const"; so "k" was chosen, since that's the first letter of "konstant" in German, and is widely used for constants in mathematics.

https://stackoverflow.com/questions/5016622/where-does-the-k-prefix-for-constants-come-from

- enumの大文字小文字はgoogle cpp style guideに沿った形かな
https://google.github.io/styleguide/cppguide.html#Enumerator_Names

Reply:
まなび！！